### PR TITLE
[AppService] Allow Azure Functions Create to accept Python 3.9 parameters

### PR DESCRIPTION
**Pull request by Hazhzeng**
**Description**

Azure Functions team is going to support Python 3.9 in December, we need to allow customers to create Python 3.9 function app in azure-cli before the Python 3.9 announcement.

This is **NOT** an official Azure Functions Python 3.9 release.

cc: @gzuber, @pragnagopa 

**Testing Guide**

Customer should be able to create Python 3.9
1. On Linux Consumption with `az functionapp create -g <resource_group> -n <app_name> -c <consumption_plan_location> -s <storage_account> --os-type linux --runtime python --runtime-version 3.9 --functions-version 3`
2. On Linux Dedicated/Premium with `az functionapp create -g <resource_group> -n <app_name> -p <dedicated_plan_name> -s <storage_account> --os-type linux --runtime python --runtime-version 3.9 --functions-version 3`

**History Notes**

[AppService] az functionapp create: accept "--runtime python" and  "--runtime-version 3.9" as Azure Functions v3 parameter

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by pragnagopa** [comment]
@Hazhzeng - This is driven by updating stacks so that all tooling can be notified of the support.

Please send PR to updates to two repos:
1. Sample PR1: https://github.com/pragnagopa/azure-functions-supported-runtime-stacks/pull/20  
2. Sample PR2 to udpate Ux: https://github.com/pragnagopa/azure-functions-supported-runtime-stacks/pull/20

File: src/azure-cli/azure/cli/command_modules/appservice/_constants.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 OS_DEFAULT = "Windows"
 STATIC_RUNTIME_NAME = "static"  # not an official supported runtime but used for CLI logic
 NODE_VERSIONS = ['10.6', '10.14']
-PYTHON_VERSIONS = ['3.8', '3.7', '3.6']
+PYTHON_VERSIONS = ['3.9', '3.8', '3.7', '3.6']
```
Comment (pragnagopa) [inline src/azure-cli/azure/cli/command_modules/appservice/_constants.py:23]: @gzuber - is this needed? versions should be coming from stacks right?

File: src/azure-cli/azure/cli/command_modules/appservice/resources/LinuxFunctionsStacks.json [hunk 2]
```diff
@@ -216,9 +216,9 @@
                             "use32BitWorkerProcess": false,
                             "linuxFxVersion": "Python|3.9"
                         },
-                        "isPreview": false,
+                        "isPreview": true,
                         "isDeprecated": false,
-                        "isHidden": false
```
Comment (pragnagopa) [inline src/azure-cli/azure/cli/command_modules/appservice/resources/LinuxFunctionsStacks.json]: Once changes to support python 3.9 are deployed to all regions in Azure, you need to send another PR to un-hide

Comment (pragnagopa) [reply src/azure-cli/azure/cli/command_modules/appservice/resources/LinuxFunctionsStacks.json]: in other words - if changes are already deployed to Azure, no need to set the `isHidden` property - just setting preview to true is enough

**Review by pragnagopa — CHANGES_REQUESTED** [review]
Please send PRs to 
- https://github.com/Azure/azure-functions-ux/
- https://github.com/pragnagopa/azure-functions-supported-runtime-stacks

**Comment by Hazhzeng** [comment]
CI fails with extension error but I haven't changed anything in extension though, please advise.
```
Please ensure you have network connection.
Error detail: HTTPSConnectionPool(host='aka.ms', port=443): Max retries exceeded
with url: /azure-cli-extension-index-v1 (Caused by SSLError(SSLError(1, '[SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:852)'),))
```

**Review by pragnagopa — APPROVED** [review]

**Review by qwordy — APPROVED** [review]

**Review by gzuber — APPROVED** [review]

**Comment by yonzhan** [comment]
AppService

**Comment by Hazhzeng** [comment]
Hi @qwordy, could you help me to re-run **Test Extensions Loading Python36** and see if it goes through. Not sure if this will block the merge? Thanks.

**Comment by qwordy** [comment]
> 
> 
> Hi @qwordy, could you help me to re-run **Test Extensions Loading Python36** and see if it goes through. Not sure if this will block the merge? Thanks.

Sure.

**Comment by Hazhzeng** [comment]
Hi @qwordy, please mark this to S179 milestone.

**Comment by qwordy** [comment]
> 
> 
> Hi @qwordy, please mark this to S179 milestone.

OK

**Pull request merged**